chore(API): Add styling to credential callback and autoclose window

dana-gill opened the pull request:
## Summary

Add styling to credential callback and autoclose window. This looks more professional and also is a small reward for users setting up credentials

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/NODE-1514/google-oauth-confirmation-modal-re-add-auto-close

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) <!--
   **Remember, the title automatically goes into the changelog.
   Use `(no-changelog)` otherwise.**
-->
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. <!--
   A bug is not considered fixed, unless a test is added to prevent it from happening again.
   A feature is not complete without tests.
-->
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

Commit:
Add styling to callback and autoclose window

ShireenMissi reviewed (commented):
LGTM 🎉
Asked for design review before merging

Commit:
Incorporate design feedback

Due to the nature of rendering handlebar files, it's a bit of a
challenge to incorporate custom font styles. Hence, we are using
sans-serif

Joffcom commented on packages/cli/templates/oauth-callback.handlebars:
```diff
@@ -1,10 +1,85 @@
+		<div class="center-container">
+			<div class="left-container">
+				<div class="row">
+					<img src='https://github.com/n8n-io/n8n/blob/master/assets/n8n-logo.png?raw=true' class="logo" />
```
Can we load this locally instead of pulling it from a remote service? If someone for some reason has GitHub blocked on an enterprise web filter or GitHub is having a hard time this could cause issues.

Commit:
Load image from relative path

Commit:
Adapt tests

Joffcom reviewed (approved):
Looks good, I am happy with this 🙌🏻

janober commented:
Got released with [`n8n@1.76.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.76.0)
